docs/content/en/docs/Development/ide.md

```diff
@@ -0,0 +1,316 @@
+---
+title: IDE configuration
+weight: 50
+---
+
+## heir-lsp
+
+HEIR provides an LSP server that extends the MLIR LSP server with HEIR's
+dialects.
+
+Build the LSP binary, then move it to a location on your path.
+
+```bash
+bazel build //tools:heir-lsp
+cp bazel-bin/tools/heir-lsp /usr/local/bin
+```
+
+Then point your IDE to the LSP in the normal manner.
```

Comment:
I don't recommend this approach, as it means you'll need to remember to re-copy the lsp whenever a new op is added, either upstream or in HEIR. What I end up doing is just pointing the IDE to the bazel-bin versions of heir-lsp.

Also, the docs should include the .vscode/settings.json snippet for this (also for buildifier location, using the ~/bin location we recommend?)

Reply:
Updated to give both options.